@uportal/content-carousel/src/App.vue

```diff
@@ -117,7 +134,7 @@
           </p>
         </div>
       </content-carousel>
-    </section>
+    </section>-->
```

Comment:
do we still need the commented out section?

Reply:
Resolved by uncommenting these, they are for testing